app/assets/javascripts/discourse/app/models/user.js

```diff
@@ -1060,9 +1075,13 @@ const User = RestModel.extend({
     );
   },
 
-  // obsolete, just call "user.timezone" instead
   resolvedTimezone() {
-    return this.timezone;
+    deprecated(
+      "You should use `user.user_option.timezone` instead of `resolvedTimezone`",
```

Comment:
Using the second perspective in the deprecation message is alittle odd. I think we can just say "resolvedTimezone has been deprecated etc... Use `user.user_option.timezone`" instead. Although I do wonder if we can just drop this entire function since there is a comment about it being obsolete. This technically isn't any public API.